src/main/java/io/jenkins/plugins/analysis/core/portlets/IssuesChartPortlet.java

```diff
@@ -127,7 +128,7 @@ public void setHeight(final int height) {
     public String getTrend() {
         SeverityTrendChart severityChart = new SeverityTrendChart();
 
-        List<Iterable<? extends AnalysisBuildResult>> histories = jobs.stream()
+        List<Iterable<BuildResult<AnalysisBuildResult>>> histories = jobs.stream()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/app/uhafner/warnings-ng-plugin/pullRequest?prid=4437947)